multi-int-store/src/main/java/com/connexta/multiintstore/config/S3StorageConfiguration.java

```diff
@@ -45,18 +43,12 @@ public AmazonS3 amazonS3(@NotNull final AmazonS3Configuration configuration) {
         .build();
   }
 
-  // TODO does this need to be a separate bean?
-  @Bean
-  @Scope(value = ConfigurableBeanFactory.SCOPE_SINGLETON)
-  public TransferManager transferManager(@NotNull AmazonS3 amazonS3) {
-    return TransferManagerBuilder.standard().withS3Client(amazonS3).build();
-  }
-
   @Bean
   public S3StorageAdaptor s3StorageAdaptor(
       @NotNull final AmazonS3 amazonS3,
-      @NotNull TransferManager transferManager,
       @Value("${aws.s3.bucket.quarantine}") @NotBlank final String bucket) {
+    TransferManager transferManager =
```

Comment:
Shouldn't this be in the `S3StorageAdaptor` constructor instead? It's an implementation detail of the `S3StorageAdaptor`, not a configuration.

Reply:
I think it's better to have Spring provide the object through Dependency Injection rather than the `S3StorageAdaptor` having a dependency directly on `TransferManager`.

Reply:
Only the implementation of S3StorageAdaptor needs TransferManager. If we decide not to use TransferManager in the future to handle storing large files, we should only need to change the S3StorageAdaptor class, not the constructor/configuration.